.github/workflows/manual-deploy-testnet-l2.yml

```diff
@@ -65,6 +65,7 @@ jobs:
           echo "WORKER_ADDR=${{vars.WORKER_ADDR}}"
           
           echo "BATCH_INTERVAL=${{vars.BATCH_INTERVAL}}"
+          echo "L2_MAX_BATCH_INTERVAL=${{vars.L2_MAX_BATCH_INTERVAL}}"
```

Comment:
This config key looks a bit different because it's using the new naming convention (got a PR in flight to update the rest).